modules/server/src/main/java/org/jboss/wsf/stack/cxf/security/authentication/SubjectCreatingInterceptor.java

```diff
@@ -59,21 +60,6 @@
  */
 public class SubjectCreatingInterceptor extends WSS4JInInterceptor
 {
-    private static final Class<?> groupClass;
-
-    static
-    {
-        Class<?> clazz = null;
-        try
-        {
-            clazz = Class.forName("java.security.acl.Group");
-        } catch (Throwable t)
-        {
-            // ignore
-        }
-        groupClass = clazz;
-    }
-
```

Comment:
Why are these lines removed ? This is for https://issues.redhat.com/browse/JBWS-4133.

Reply:
changed